devito/operator/operator.py

```diff
@@ -219,6 +219,12 @@ def _build(cls, expressions, **kwargs):
 
         # Required for the jit-compilation
         op._compiler = kwargs['compiler']
+
+        # Required for compilation by the profiler
```

Comment:
Why aren't those part of the profller init?

Reply:
It was before, and this was the reason it was not working, as a different compiler was build after doing the profiler init. Related discussion here: https://devitocodes.slack.com/archives/C7JMLMSG0/p1739458707616429?thread_ts=1739458681.588539&cid=C7JMLMSG0